Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM ramsproject/rams
+FROM vmearl/rams:magfest-merge
 
 ADD src plugins/
 ADD sideboard-development.ini ./development.ini
 RUN /app/env/bin/paver install_deps
 
-# derp, fix this to be done in sideboard instead
-CMD /app/env/bin/python3 /app/sideboard/run_server.py
+ADD uber-wrapper.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/uber-wrapper.sh
```

Comment:
This can probably be cleaned up. Either we can fix the permissions on the script before we load it up or we can do this in an && with the paver dependency add.